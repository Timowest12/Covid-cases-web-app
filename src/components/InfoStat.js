import React from 'react';

const InfoStat = (props) => (
  <div className="infoStatsBlock">
    <div className="infoStat">
      <div className="infoStatLeft">{props.metric}</div>
      <div className="infoStatright">{props.amount}</div>
    </div>
  </div>

);

export default InfoStat;
