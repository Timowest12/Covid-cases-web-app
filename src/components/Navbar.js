import React from 'react';
import { BsChevronLeft } from 'react-icons/bs';
import { Link } from 'react-router-dom';

const Navbar = (props) => {
  const { page } = props;
  return (
    <nav>
      {page === 'Country data' ? <Link to="/"><div className="left"><BsChevronLeft /></div></Link> : <div />}
      <div className="right">{page}</div>
      <div className="right">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</div>
    </nav>
  );
};

export default Navbar;
