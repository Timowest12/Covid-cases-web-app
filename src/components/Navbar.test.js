import React from 'react';
import renderer from 'react-test-renderer';
import InfoStat from './InfoStat';
import Navbar from './Navbar';

it('Test if navbar renders correctly', () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
