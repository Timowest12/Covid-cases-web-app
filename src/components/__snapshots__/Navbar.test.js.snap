// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if navbar renders correctly 1`] = `
<nav>
  <div />
  <div
    className="right"
  />
  <div
    className="right"
  >
          
  </div>
</nav>
`;
